### DISCUSSION TITLE
LeanCloud 国际版政策变化是否会影响大陆用户访问部署在境外的服务

### DISCUSSION THREAD

**Discussion by Nemo1166** [walinejs/waline#1204] [Q&A] [answered]
下午我也收到了 [这个discussion](https://github.com/walinejs/waline/discussions/1203) 相同内容的邮件。

目前个人项目的部署情况：

- 主服务(基于 vuepress)：部署于 vercel
- waline：部署于vercel，使用 vercel.app 子域名，serverless 函数部署于中国香港

目前大陆用户访问正常，我是否需要为 LeanCloud 即将到来的政策变化采取额外措施？

**Comment by lizheming** [comment, marked as answer]
LeanCloud 国际版 IP 限制的这个事情，对 Waline 没有影像哈，Waline 是用 Vercel 访问的 LeanCloud，Vercel 是国外 IP 所以不受影响，
